### ISSUE TITLE
Lodash is not tree shaked

### ISSUE THREAD

**Issue by alinaMihai** [azizali/react-super-treeview#18]
Hi,
Great library ! 
Would it be possible to adjust the configuration so that Lodash is tree shaken correctly ?
From what I've noticed the distribution uses require module syntax and the whole of Lodash is imported. 76.1% of the code size is just lodash https://bundlephobia.com/result?p=react-super-treeview@0.1.0,

It would be so great if this could be tree shaken. 

Thank you!

**Comment by azizali** [comment]
Hey @alinaMihai any suggestions on how this can be accomplished. I see that this can be done with Webpack 4.0, any thoughts?

**Comment by alinaMihai** [comment]
Hi @azizali ,
I can think of two ways of doing this:
1. replacing `import { isNil, isEmpty, isEqual, find, get, cloneDeep } from 'lodash';` with

```
import isNil from 'lodash/isNil';
import isEmpty from 'lodash/isEmpty';
import isEqual from 'lodash/isEqual';
import find from 'lodash/find';
import get from 'lodash/get';
import cloneDeep from 'lodash/cloneDeep';
```

which will  import only the functionality you need in the final bundle. 

2. use [lodash-es](https://www.npmjs.com/package/lodash-es)
and change the es2015 preset in babelrc to have modules false: 
` "presets": [ [ "es2015", { "modules": false } ], "stage-0", "react"],`
Adding {"modules": false} should prevent ES6 modules from being transformed to commonJS modules, giving webpack its tree-shaking ability!

**Comment by azizali** [comment]
Nice,

The first option didn't work for me.

I even tried lodash-es but without the webpack change.

I'll take a look.

Thanks.

If you can try and post a pull request I'll take a look.

**Comment by alinaMihai** [comment]
Hi @azizali ,

I tried the first method with import for each method, and I ended up with only 31.66 kb gzipped size for lodash in my bundle, whereas before I had 124 kb gzipped.

Also if you look in the distribution you will see that it no longer exports the whole library like 
```
/***/ }),
/* 4 */
/***/ (function(module, exports) {

module.exports = require("lodash");

/***/ }),
```
but instead there is one export for each one 
```
/***/ }),
/* 4 */
/***/ (function(module, exports) {

module.exports = require("lodash/isNil");

/***/ }),
/* 5 */
/***/ (function(module, exports) {

module.exports = require("lodash/isEmpty");

/***/ }),
/* 6 */
/***/ (function(module, exports) {

module.exports = require("lodash/isEqual");

/***/ }),
/* 7 */
/***/ (function(module, exports) {

module.exports = require("lodash/find");

/***/ }),
/* 8 */
/***/ (function(module, exports) {

module.exports = require("lodash/get");

/***/ }),
/* 9 */
/***/ (function(module, exports) {

module.exports = require("lodash/cloneDeep");
```

So what do you think?

**Comment by alinaMihai** [comment]
@azizali I've opened a pull request for this. Would it be possible for you to take a look at it :) 

Thank you!

**Comment by azizali** [comment]
will do thanks so much

**Comment by azizali** [comment]
Resolved in v0.3.3.

Thanks @alinaMihai for the contribution

**Comment by alinaMihai** [comment]
Thank you @azizali  for considering it !
